rrbot:
    # publish the states of all joints
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50
    
    # position contollers
    joint1_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint1
        pid: {p: 100.0, i: 0.01, d: 10.0}
    joint2_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint2
        pid: {p: 100.0, i: 0.01, d: 10.0}

    # this is an extra joint that doesnt exist for the original rrbot
    joint3_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint3
        pid: {p: 100.0, i: 0.01, d: 10.0}

    